From 7b2b140df13d7e2a49b5cd25e44c4fde4e626170 Mon Sep 17 00:00:00 2001
From: jinghua <xigu@marvell.com>
Date: Fri, 26 Aug 2016 17:03:40 +0800
Subject: [PATCH 0945/2241] dts: a3700: update sdio gpio of voltage switch for
 rev2 devel-board

Armada3700 devel-board rev1 uses gpio 22 for sdio voltage switch
while the rev2 board changes it to gpio 23. This patch updated
the gpio reference in rev2 board's dts file.

Change-Id: I23ceeeb7bba6dc427d77e2d3f8edd68449adcb0f
Signed-off-by: jinghua <xigu@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/32432
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Tested-by: Wilson Ding <dingwei@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-3720-db.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-3720-db.dts b/arch/arm64/boot/dts/marvell/armada-3720-db.dts
index 19429f9..00f89bb 100644
--- a/arch/arm64/boot/dts/marvell/armada-3720-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3720-db.dts
@@ -176,7 +176,7 @@
 				regulator-max-microvolt = <3300000>;
 				regulator-boot-on;
 
-				gpios = <&gpio_sb 22 GPIO_ACTIVE_HIGH>;
+				gpios = <&gpio_sb 23 GPIO_ACTIVE_HIGH>;
 				gpios-states = <0>;
 				states = <1800000 0x1
 					  3300000 0x0>;
-- 
2.7.4

